pkg/api/getambassador.io/v3alpha1/authservice_types.go

```diff
@@ -34,7 +34,7 @@ type AuthServiceIncludeBody struct {
 
 // Why isn't this just an int??
```

Comment:
Probably should drop this comment sometime. 😂  (Also, thank you. 🙂 )

Reply:
No, that comment is still valid. But I'll reword it to be clearer.